CMakeLists.txt

```diff
@@ -28,6 +28,33 @@ if(NOT CMAKE_BUILD_TYPE)
   message(STATUS "Build type not specified: Use Release by default.")
 endif()
 
+#------------------------------------------------------------------------
+# Static Windows Runtime
+#   Option to statically link to the Windows runtime. Maybe only 
+#   applies to WIN32/MSVC.
+#------------------------------------------------------------------------
+if (MSVC)
+    option( USE_STATIC_RUNTIME "Set ON to change /MD(DLL) to /MT(static)" OFF )
```

Comment:
Just a comment: This should definitely stay `OFF` via default. Overriding runtimes by default is annoying. Unfortunately, many project do this